Builds/Test.py

```diff
@@ -155,6 +155,13 @@ def powerset(iterable):
 )
 
 parser.add_argument(
+    '--testjobs', '-j',
```

Comment:
@ximinez What do you think of removeing the short `-j` option (but leaving the long option)? Rational: `Test.py -j4` and `Test.py -- -j4` do very different things and having a -`j` option here make it easy to make a mistake.

While we're at it, I think we should make both program use the same option names. I prefer `unittest-jobs`, but I'm OK renaming the one in `rippled` to `testjobs` if you disagree.

Reply:
I'm not ximenez, but I agree that `-j` is confusing and I also thought about requesting its removal.

Reply:
I'm fine with removing the short `-j` option from `Test.py`, but I don't think it's really necessary to have the options match between the utility and `rippled`. Justification: 1) we've already got `--test` to specify which test is passed to `--unittest`. 2) In `Test.py`, unit testing is implied and always done, so we don't need to be as explicit in the option names as `rippled`, where testing is _not_ the default and there are many other things it can do.

Reply:
Fixed